packages/applications/legacy/src/controllers/project/getProjectPage/_utils/getReprésentantLégal.ts

```diff
@@ -38,17 +39,20 @@ export const getReprésentantLégal: GetReprésentantLégal = async (identifiant
       });
 
     if (Option.isSome(représentantLégal)) {
+      const featureActivated = isDemandeChangementReprésentantLégalEnabled();
+
       const demandeChangementExistante = await getChangementReprésentantLégal(identifiantProjet);
 
       const peutConsulterLaDemandeExistante =
-        demandeChangementExistante && utilisateur.aLaPermission('représentantLégal.consulter');
+        utilisateur.aLaPermission('représentantLégal.consulter') && demandeChangementExistante;
 
       const peutFaireUneDemande =
-        !demandeChangementExistante &&
-        utilisateur.aLaPermission('représentantLégal.demanderChangement');
+        featureActivated &&
+        utilisateur.aLaPermission('représentantLégal.demanderChangement') &&
+        !demandeChangementExistante;
 
       const peutFaireModification =
```

Comment:
Une modification c'est peut être pas assez explicite

```suggestion
      const peutModifier =
```